packages/holocron/src/HolocronErrorBoundary.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { node } from 'prop-types';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+
+class HolocronErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { error: null };
+  }
+
+  static getDerivedStateFromError(error) {
+    return { error };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    this.setState({
+      error,
+    });
+    console.error(error);
+    console.error(errorInfo);
+  }
+
+  render() {
+    const { props: { children }, state: { error } } = this;
+
+    if (error) {
+      if (process.env.NODE_ENV === 'production') return null;
+      return <div style={{ color: 'red' }}>{error.message}</div>;
+    }
+
+    return children;
```

Comment:
this won't work if there are multiple children

Reply:
Taken care of in 42a0ed6